scripts/printProcessQueue.py

```diff
@@ -76,6 +87,26 @@ def output_text(items):
 
     dbu = DButils.DButils(options.database)
     items = dbu.ProcessqueueGetAll()
+     
+    # Check if the processque is empty 
+    # Usually used for bash scripts
+    #   return 0 if data exists (true)
+    #   return 1 if processque is empty (false)
+    if( options.exist ):
+        if( len(items) != 0 ): 
+            print("Process Queue is not empty\n" if not options.quiet else "", end='')
+            del dbu
+            sys.exit(0)
+        else:
+            del dbu
+            sys.exit(1)
+
+    # return the number of items in processqueue 
+    if( options.count ):
+         print("Number of items in processqueue %d\n" % len(items) if not options.quiet else "", end='')
+         del dbu
+         sys.exit(len(items))
```

Comment:
I think this needs to be special-cased. The return code is eight bit and it looks to just take the low bits...so if there are 256 items, the value will be 0. Maybe use 255 if there are more than 255, so it just indicates "lots"?

It looks like Python also uses 1 as the return code for an uncaught exception, so there's no distinguishing between "uncaught exception" and "one item in the process queue." I guess to some extent the user asked for it, so they deserve what they get? This also applies to the simple "exist" case. I checked and `grep` uses 1 for not found, 2 for error, so the cases can be distinguished. Maybe the short-term solution here is just to shrug our shoulders, say "not perfect", and document.

Reply:
I can check for more than 255, I don't really need to use 1 I can return another error code that make more sense.

Reply:
Special-cased >255, documented the 1 case.